saleor/attribute/__init__.py

```diff
@@ -5,15 +5,17 @@ class AttributeInputType:
     MULTISELECT = "multiselect"
     FILE = "file"
     REFERENCE = "reference"
+    TEXT = "text"
 
     CHOICES = [
         (DROPDOWN, "Dropdown"),
         (MULTISELECT, "Multi Select"),
         (FILE, "File"),
         (REFERENCE, "Reference"),
+        (TEXT, "Text"),
     ]
     # list of the input types that can be used in variant selection
-    ALLOWED_IN_VARIANT_SELECTION = [DROPDOWN]
+    ALLOWED_IN_VARIANT_SELECTION = [DROPDOWN, TEXT]
```

Comment:
Are we sure that we want to use "text" in variant selection? When an attribute is used in variant selection, it will be required to choose by the customer in the storefront to identify a variant; it will also appear in the variant creator in the dashboard and staff users will have to provide all text values for all variants to create them.

![image](https://user-images.githubusercontent.com/5421321/112823641-d4cc2500-9089-11eb-8493-e38f3c5de99b.png)

Reply:
Good point 👍